roles/rundeck/tasks/rundeck-cli.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Download the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' wget {{ rundeck_cli_url }}
+
+- name: Unpack the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' sudo unzip -qqo {{ rundeck_cli_url | basename }} -d {{ rundeck_cli_container_path }}
+
+- name: Create symlink to Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' sudo ln -sf {{ rundeck_cli_container_path }}/{{ rundeck_cli_url | basename | splitext | first }}/bin/rd {{ rundeck_cli_container_path }}/rd
+
+- name: Clean up Rundeck CLI artifact inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' rm -f {{ rundeck_cli_url | basename }}
+
+- name: Add alias for Rundeck CLI
+  ansible.builtin.lineinfile:
+    path: '{{ ansible_env.HOME }}/.bashrc'
+    create: yes
+    line: "alias rd='podman exec -it '{{ rundeck_container_name }}' rd'"
```

Comment:
We want containers that disappear, not remain there

Reply:
The container is already there and running as a server

Reply:
You are missing the point. When the service is stopped then the container should vanish, which means that anything that was done inside of the container disappears too. Mount the JAR files

Reply:
I switched to mounting yesterday